indico/modules/categories/serialize.py

```diff
@@ -101,6 +101,7 @@ def serialize_category(category, with_favorite=False, with_path=False, parent_pa
         'title': category.title,
         'is_protected': category.is_protected,
         'has_events': category.has_events,
+        'has_subcategories': bool(category.children),
```

Comment:
Better add `has_children` similar to `has_events` (make sure to undefer it as well when querying) - no need to query all the subcategories if we can just get the count

Reply:
Thanks for the help :) I think it should be good now :+1: